.github/workflows/test-coverage.yml

```
@@ -0,0 +1,120 @@
+              body: comment
+            });
+          }
+
+  coverage-gate:
+    needs: test
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+
+    steps:
+    - name: Download Coverage Reports
+      uses: actions/download-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/download-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)